### DISCUSSION TITLE
[Suggestion] In-game FPS limiter

### DISCUSSION THREAD

**Discussion by Kaiser-Indrasil** [tsunamayo/Starship-EVO#3947] [Suggestions]
Hey, Tsuna!

Hopefully moving houses is going smoothly.

Meanwhile, I wanted to ask for a feature that would help me out and hopefully other people, too, in a couple of ways.
It's an in-game FPS limiter.

Since Starship EVO is designed in a way to maximise GPU usage to produce maximum performance, oftentimes people's GPUs (including mine) run it at 100% capacity.
On one hand, this is what you want, since the game won't hiccup when you charge it with a lot of calculations.
On another, it's not optimal, since most of the time we're just building away, which produces very little calculations to perform.
Ideally, this could be alleviated by optimising the game in a way that limits resource usage to the point where the game takes up only as much computational power as it needs in any given moment.
However, we're still years from that point and that's fine.
This is where the FPS limiter would come in handy.
Instead of running the game at 100 FPS like I've been to date, I could run it at 45 and let the VSync/GSync smooth the frames together.
Sure, I can do it through the Nvidia Control Panel but it's tedious... I'd much rather be able to do it in game.

Second use for this feature would be to synchronise the framerates when I'm taking footage from the game.
For example, when the game runs at 100 FPS and I record at 60 (because YouTube accepts up to 60 FPS only), there will be noticeable periodic micro stutters.

**Comment by Uncle-Ulty** [comment]
That was the first thing I noticed just after installing this game. I made the Nvidia Control setting cuz my GPU was running like crazy.

**Comment by tsunamayo** [comment]
Well there is the v-sync already, so I guess you want something where you can specify the FPS precisely? Or maybe running at half-rate would be enough? Thanks

**Comment by Kaiser-Indrasil** [comment]
Yeah, I would love to adjust FPS manually, so that my PC can run the game at lower than oven-like temperatures 😅
